app/components/works/version_component.rb

```diff
@@ -0,0 +1,13 @@
+# typed: true
+# frozen_string_literal: true
+
+module Works
+  # Renders a widget for setting the title and contact for the work.
+  class VersionComponent < ApplicationComponent
+    def initialize(form:)
+      @form = form
+    end
+
+    attr_reader :form
```

Comment:
Should the component render no matter what state the work is in? Or should it only render in certain states? Might be a PO question. cc: @amyehodge @jcoyne

Reply:
That's a good question. I figured as this is the edit form it probably always needs to be there... but validation/etc?

Reply:
It needs to be there for VERSION drafts, but not for the initial drafts. In other words, it should always be there for drafts of previously accessioned content. 

A user may open this draft and save it multiple times, so 1) any entry should be saved when the draft is saved and continue to be editable until the deposit is made and 2) be required to make the deposit.